src/_playground/documentation/DocsTile/DocsTile.js

```diff
@@ -1,30 +1,56 @@
 import { Button } from '../../../Button/Button';
+import classnames from 'classnames';
 import { googlecode } from 'react-syntax-highlighter/styles/hljs';
 import PropTypes from 'prop-types';
 import SyntaxHighlighter from 'react-syntax-highlighter';
+import { Toggle } from '../../../Toggle/Toggle';
 import React, { Component } from 'react';
 
-export const DocsTile = props => {
-    const centerStyle = {
-        textAlign: 'center'
+export class DocsTile extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            backgroundToggle: false
+        };
+    }
+
+    handleToggle = () => {
+        this.setState({
+            backgroundToggle: !this.state.backgroundToggle
+        });
     };
 
-    const {centered, children} = props;
+    render() {
+        const {centered, children} = this.props;
 
-    return (
-        <div className='frDocs-Content__tile'>
-            {centered
-                ? (
-                    <div className='fd-tile__content'>
-                        <div style={centerStyle}>{children}</div>
-                    </div>
-                )
-                : (
-                    <div className='fd-tile__content'>{children}</div>
-                )}
-        </div>
-    );
-};
+        const outerDivClasses = classnames(
+            'frDocs-Content__tile',
+            {
+                'frDocs-Content__tile-background': !this.state.backgroundToggle
+            }
+        );
+
+        const innerDivClasses = classnames(
+            'fd-tile__content',
+            {
+                'frDocs-tile__centered': centered
+            }
+        );
+
+        return (
+            <div className={outerDivClasses}>
+                <Toggle
+                    className='frDocs-tile__background-toggle'
+                    inputProps={{'aria-label': 'Toggle background color'}}
+                    onChange={this.handleToggle}
+                    size='xs' />
```

Comment:
I think we should add "Toggle background" as the `children` for `Toggle` so it's clear _what_ this toggle is for?  And now looking at the docs site, maybe add `children` as a prop explaining it's the label. (Or maybe #322 already is doing this?)

Reply:
perhaps add text too (right now it is toggle background - in the future it should be dark mode):

![image](https://user-images.githubusercontent.com/5056972/52012906-a8f48b00-2490-11e9-8cad-dc35d5b3f5b6.png)

Reply:
I think https://github.com/SAP/fundamental-react/issues/321 should handle the label confusion as well